.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+php:
+  - 7.1
+
+cache:
+  directories:
+    - ~/.composer/cache
+    - $HOME/.npm
+
+env:
+  - ELASTIC_SEARCH_INDEX_PREFIX=""
+
+before_install:
+  - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - phpenv config-rm xdebug.ini || return 0
+
+install:
+  - composer global require hirak/prestissimo
+  - COMPOSER_MEMORY_LIMIT=-1 travis_retry composer install --optimize-autoloader --no-interaction --no-progress --dev
```

Comment:
Also, Im just thinking here and maybe it is not good idea but:

We could cache a vendor folder too, we do not keep composer.lock file anyway so it will update dependencies everytime right? And we got few seconds faster travis build

Reply:
I do not like cache final results of `vendor` or `node_modules` folders, because it can make a problems. I found related issue https://github.com/travis-ci/travis-ci/issues/4579